### ISSUE TITLE
MoldUDP64 perf-test

### ISSUE THREAD

**Issue by jheusser** [paritytrading/nassau#48]
A MoldUDP64 performance test would be interesting. Might implement this.

**Comment by jvirtanen** [comment]
@jheusser Yes, it would be good to have, and it would be great if you have some time to look into that. 👍
